panther_gpiod/README.md

```diff
@@ -0,0 +1,3 @@
+# panther_gpiod
+
+Package comprises a wrapper implementation for the GPIOD library, enabling real-time manipulation of GPIO pins on the Raspberry Pi. Offering convenient interfaces for setting pin values, altering their direction, monitoring events, and conducting other GPIO operations, this library facilitates effective GPIO pin management on the Panther robot. It simplifies integration within robotic applications.
```

Comment:
Maybe also usage example in readme would be good (info about gpiomonitor and EdgeEventCallback)